projects/core/src/global-message/store/effects/global-message.effect.ts

```diff
@@ -17,7 +17,7 @@ export class GlobalMessageEffect {
   > = this.actions$.pipe(
     ofType(GlobalMessageActions.ADD_MESSAGE),
     pluck('payload', 'type'),
-    mergeMap((type: GlobalMessageType) => {
+    switchMap((type: GlobalMessageType) => {
```

Comment:
Since we want each message to be removed after a delay, wouldn't it be best to use a `concatMap` to queue the removal requests?

Reply:
Yes, I think we can use `concatMap` instead.